Source/Core/DolphinQt/Settings/PathPane.cpp

```diff
@@ -122,6 +122,12 @@ void PathPane::BrowseWFS()
   }
 }
 
+void PathPane::ResetGecko()
+{
+  Config::SetBase(Config::MAIN_GECKO_URL, "");
+  m_gecko_edit->setText(QString::fromStdString(""));
```

Comment:
```suggestion
  m_gecko_edit->clear();
```

should make this a little bit more concise